xpi-manifest.yml

```diff
@@ -111,3 +111,12 @@ xpis:
       - web-ext-artifacts/china-newtab.xpi
     addon-type: mozillaonline-privileged
     install-type: npm
+  - name: login-study
+    description: Login Study
+    repo-prefix: loginstudy
+    active: true
+    private-repo: true
+    artifacts:
+      - web-ext-artifacts/login-study.xpi
```

Comment:
We'll probably need to change package.json to rename your xpi. See how we use `mv` to rename [here](https://github.com/mozilla-extensions/normandy-devtools/blob/master/package.json#L9-L13).

Once web-ext cuts a new release with https://github.com/mozilla/web-ext/pull/1900 , we'll be able to use `web-ext --filename`.

Reply:
Ah, looks like we could change [these lines](https://github.com/mozilla-extensions/login-study/blob/master/build.js#L128-L131) since build.js already renames.